modules/planning/proto/planning_internal.proto

```diff
@@ -159,6 +153,7 @@ message OpenSpaceDebug {
   optional apollo.planning_internal.Trajectories fallback_trajectory = 15;
   optional apollo.common.TrajectoryPoint trajectory_stitching_point = 16;
   optional apollo.common.TrajectoryPoint future_collision_point = 17;
+  optional double time_latency = 18; // ms
```

Comment:
please use more trial name, like `time_latency_ms`?

Reply:
In that case, if we decide to use sec as unit in future, we will have to rename this field.